src/pages/ErrorPage.vue

```diff
@@ -1 +1,62 @@
-<template><h1>Error Page</h1></template>
+<template>
+  <div class="container">
+    <h1>404</h1>
+    <h2>Page not found</h2>
+    <h3>The Page you are looking for doesn't exist !</h3>
```

Comment:
This is not a heading really, use a `div` or `p` tag
```suggestion
    <div>The Page you are looking for doesn't exist !</div>
```